ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/UNDERLAY-MULTICAST-L3LEAF1A.cfg

```diff
@@ -36,7 +36,6 @@ interface Port-Channel6
    switchport
    switchport trunk allowed vlan none
    switchport mode trunk
-   mlag 6
```

Comment:
Add a test for `mlag_on_orphan_port_channel_downlink: true` condition.

Reply:
Adjust the old test for the false case to be testing the true case instead.